src/flexalloc_xnvme_env.h

```diff
@@ -32,7 +33,11 @@ fla_xne_sync_seq_w_naddrs(struct xnvme_dev * dev, const uint64_t slba, const uin
 
 int
 fla_xne_sync_strp_seq_x(struct xnvme_dev *dev, const uint64_t offset, uint64_t nbytes,
```

Comment:
Can we get rid of this function? It looks like it isn't being used anymore.

Reply:
I left it there because I was using it for performance comparison. This of course can be done by comparing my branch with master. Will remove it so we don't forget. thx.

Reply:
Done